packages/core/src/accessibility/focusStyleManager.ts

```diff
@@ -9,7 +9,14 @@ import { InteractionModeEngine } from "../common/interactionMode";
 
 export const FOCUS_DISABLED_CLASS = "pt-focus-disabled";
 
-const focusEngine = new InteractionModeEngine(document.documentElement, FOCUS_DISABLED_CLASS);
+const fakeFocusEngine = {
+    isActive: () => true,
+    start: () => true,
+    stop: () => true,
+};
+const focusEngine = typeof document !== "undefined"
+    ? new InteractionModeEngine(document.documentElement, FOCUS_DISABLED_CLASS)
+    : fakeFocusEngine;
 
 // this is basically meaningless to unit test; it requires manual UI testing
 /* istanbul ignore next */
```

Comment:
Sounds good, thanks. I was just struggling with that. Coverage inexplicably runs on CircleCI but not locally, and I can't figure out why.